book/listings/todo/8/window/mod.rs

```diff
@@ -397,46 +397,37 @@ impl Window {
 
     // ANCHOR: new_collection
     fn new_collection(&self) {
-        // Create new Dialog
-        let dialog = Dialog::with_buttons(
-            Some("New Collection"),
-            Some(self),
-            DialogFlags::MODAL
-                | DialogFlags::DESTROY_WITH_PARENT
-                | DialogFlags::USE_HEADER_BAR,
-            &[
-                ("Cancel", ResponseType::Cancel),
-                ("Create", ResponseType::Accept),
-            ],
-        );
-        dialog.set_default_response(ResponseType::Accept);
-
-        // Make the dialog button insensitive initially
-        let dialog_button = dialog
-            .widget_for_response(ResponseType::Accept)
-            .expect("The dialog needs to have a widget for response type `Accept`.");
-        dialog_button.set_sensitive(false);
+        let cancel_response = "cancel";
+        let create_response = "create";
```

Comment:
I feel this should be declared after `Entry`

Reply:
Fixed by 48572ed228fff0d426e56f1cfe409918ecf03d1b